starter_project_mobile_2022/lib/profile_page_dave/presentation/profile_page.dart

```diff
@@ -0,0 +1,307 @@
+                    child: Image.network(
+                        "https://www.princeton.edu/sites/default/files/styles/half_2x/public/images/2018/12/IMG_9127.jpg?itok=156LUKD5",
+                        loadingBuilder: (BuildContext context, Widget child,
+                            ImageChunkEvent? loadingProgress) {
+                      if (loadingProgress == null) {
+                        return child;
+                      }
+                      return Center(
+                        child: CircularProgressIndicator(
+                          value: loadingProgress.expectedTotalBytes != null
+                              ? loadingProgress.cumulativeBytesLoaded /
+                                  loadingProgress.expectedTotalBytes!
+                              : null,
+                        ),
+                      );
+                    },
+                        fit: BoxFit.cover,
+                        width: width * 0.28,
+                        height: height * 0.3),
+                  ),
+                  Padding(
+                    padding: EdgeInsets.only(
+                        left: height * 0.23, top: height * 0.24),
+                    child: Positioned(
+                      bottom: 0,
+                      right: height * 0.001,
+                      child: buildEditIcon(edit_color),
+                    ),
+                  ),
+                ]),
+              ),
+              SizedBox(width: height * 0.08),
+              Column(
+                crossAxisAlignment: CrossAxisAlignment.start,
+                children: [
+                  Text(
+                    'Nati BC',
+                    maxLines: 1,
+                    style: TextStyle(
+                      fontSize: width * 0.05,
+                      color: black,
+                      fontWeight: FontWeight.bold,
+                    ),
+                  ),
+                  Text('UX designer', maxLines: 1),
+                ],
+              )
+            ],
+          ),
+          Padding(
+            padding: EdgeInsets.only(bottom: width * 0.08),
+            child: Row(
+              crossAxisAlignment: CrossAxisAlignment.end,
+              mainAxisAlignment: MainAxisAlignment.end,
+              children: [
+                Column(
+                  crossAxisAlignment: CrossAxisAlignment.start,
+                  children: const [
+                    Text("About me", maxLines: 1),
+                    Text("I’m a a software engineer and UX designer",
+                        maxLines: 1, overflow: TextOverflow.ellipsis),
+                  ],
+                )
+              ],
+            ),
+          ),
+        ],
+      ),
+    );
+  }
+}
+
+Widget buildEditIcon(Color color) => buildCircle(
+      color: white,
+      all: 3,
+      child: buildCircle(
+        color: color,
+        all: 8,
+        child: const Icon(
+          Icons.edit,
+          color: white,
+          size: 10,
```

Comment:
Use following code instead of hard-coded size variables.
`double width = MediaQuery.of(context).size.width`
`...`
`size: width * 0.01` etc.